app/models/monthly_report.rb

```diff
@@ -99,6 +99,18 @@ def browseable?(other_user)
     shipped? || user == other_user
   end
 
+  def self.target_month_select_options
+    options = []
+    target_months = self.released.pluck(:target_month).sort.uniq
+    first_month, last_month = target_months.first, target_months.last
```

Comment:
__[rubocop]__
* Do not use parallel assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#parallel-assignment, https://github.com/JuanitoFatas/fast-ruby#parallel-assignment-vs-sequential-assignment-code)

<!-- Comment by [SideCI](https://sideci.com) -->